src/views/overview/index.vue

```diff
@@ -0,0 +1,299 @@
+<template>
+  <div class="display-box">
+    <h1 class="title">
+      <span>Protein </span>
+      <span
+        v-for="(s, index) in font"
+        :key="index"
+        v-bind:style="{ color: fontColor[index] }"
+        >{{ s }}</span
+      >
+    </h1>
+    <!-- serch -->
+    <div class="search-box">
+      <el-select
+        v-model="sapiens"
+        class="m-2"
+        placeholder="Select organism"
+        style="width: 140px"
+      >
+        <el-option
+          v-for="item in options"
+          :key="item.value"
+          :label="item.label"
+          :value="item.value"
+        />
+      </el-select>
+      <el-input
+        v-model="input"
+        placeholder="protein name"
+        clearable
+        class="input"
+      />
+      <el-button type="primary" plain @click="queryProtein(input)"
+        >Search</el-button
+      >
+    </div>
+    <p>
+      Find the iBAQ distribution for your protein of interest, e.g. ABCD4_HUMAN.
+    </p>
+    <div class="info">
+      <h1>Organism: <span>Homo sapiens</span></h1>
+      <h1>
+        Protein: <span>{{ protein }}</span>
+      </h1>
+    </div>
+    <!--  Image  -->
+    <div class="card">
+      <div ref="chart" style="width: 100%; height: 600px"></div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import * as echarts from 'echarts'
+import { ref, onMounted, reactive, toRaw } from 'vue'
+import { getProteins } from '@/api/search'
+import { inflate } from 'pako'
+// sapiens
+const sapiens = ref('')
+// select_menus
+const options = [
+  {
+    value: 'Homo sapiens',
+    label: 'Homo sapiens'
+  }
+]
+// 加载页面时请求表单
+onMounted(() => {
+  getProteinTable()
+})
+// 获取完整表单
```

Comment:
Please refrain from using non-English comments. This applies here, but also in other places. Non-English comments are useless for me. I will not go and translate each individual comment to understand whats written. And neither will other people who might be able to help.